packages/shopify-codemod/transforms/utils.js

```diff
@@ -1,4 +1,63 @@
 import j from 'jscodeshift';
+import chalk from 'chalk';
+
+function range(file, node, source, color, CONTEXT = 2) {
+  if (file === null || node === null || source === null || color == null) {
+    return '';
+  }
+
+  const lines = source.split(/\r\n|[\n\v\f\r\x85\u2028\u2029]/);
```

Comment:
Can you move the regex outside the function and give it a meaningful variable name?